packages/wrangler/src/metrics/metrics-config.ts

```diff
@@ -20,7 +15,6 @@ import { getAPIToken } from "../user";
  * gathering.
  */
 export const CURRENT_METRICS_DATE = new Date(2022, 6, 4);
```

Comment:
Since we're no longer interactively prompting we should probably remove this re-prompting logic

Reply:
thought there wasn't much harm in recording the date, in case in the future we make updates and do want to print a notice or something. don't feel strongly about it though